cmd/kube-scheduler/app/options/options.go

```diff
@@ -200,6 +216,12 @@ func (o *Options) Validate() []error {
 
 // Config return a scheduler config object
 func (o *Options) Config() (*schedulerappconfig.Config, error) {
+	if o.SecureServing != nil {
```

Comment:
no nil check needed here, MaybeDefaultWithSelfSignedCerts no-ops if SecureServing is nil

Reply:
fixed

Reply:
I appear to have led you astray... sorry:

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1b607db]

goroutine 86 [running]:
testing.tRunner.func1(0xc00030b200)
	GOROOT/src/testing/testing.go:792 +0x6a7
panic(0x1d2c320, 0x2fd61e0)
	GOROOT/src/runtime/panic.go:513 +0x1b9
k8s.io/kubernetes/cmd/kube-scheduler/app/options.(*Options).Config(0xc0001b5c80, 0xc000073e01, 0x41d0f8, 0x568f5b)
	cmd/kube-scheduler/app/options/options.go:219 +0xcb
```